package.json

```diff
@@ -9,7 +9,8 @@
     "prepare": "pnpm compile",
     "pretest": "pnpm compile",
     "posttest": "pnpm lint",
-    "publish-cli-utils": "pnpm publish build/packages/solana-cli-utils"
+    "publish-cli-utils": "pnpm compile && pnpm publish build/packages/solana-cli-utils",
+    "publish-cli-test-utils": "pnpm compile && pnpm publish build/packages/solana-cli-test-utils"
```

Comment:
Test utils will also be used for SDK testing (MintHelper is an example of class used for this). Please remove -cli- part of the name